doc/mgr/restful.rst

```diff
@@ -110,6 +110,14 @@ You can use ``curl`` in order to test your user with the API. Here is an example
 
 In the case above, we are using ``GET`` to fetch information from the ``server`` endpoint.
 
+To simplify the operation, you can ignore API user authentication with the following command::
```

Comment:
how about adding this as an option (as listed by `MODULE_OPTIONS`) instead of using two extra commands?

Reply:
I think the difference is that `option` needs to restart the `restful` module to take effect, but `extra command` can take effect immediately. which one do you think is better?

Reply:
> I think the difference is that `option` needs to restart the `restful` module to take effect

not quite true, a mgr module is able to get notified on config change by implementing the `config_notify()` method.

Reply:
If so, using `option` instead of `extra command` would be a good choice. I will modify it.